src/components/TipCard/TipCard.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import './TipCard.css'
+
+const Tip = ({ id, title, description, mod, upvotes, date}) => {
```

Comment:
I think this might be more semantic/clear if we renamed this TipCard and changed the export default to TipCard